docs/hr/content/docs/fundamentals/design.md

```diff
@@ -8,7 +8,7 @@ sidebar_position: 2
 
 Here is a schematic of the `superduperdb` design.
 
-![](/img/light.png)
```

Comment:
This isn't broken. Please test with the development server `npm run start`

Reply:
It's broken on GitHub. see: https://github.com/SuperDuperDB/superduperdb/blob/main/docs/hr/content/docs/fundamentals/design.md